src/monopoly/monopoly_player.js

```diff
@@ -75,13 +71,15 @@ MonopolyPlayer.prototype.owesRent = function(property) {
 };
 
 MonopolyPlayer.prototype.owns = function(property) {
-  return property.owner === this;
+  return property.owner && property.owner.id === this.id;
```

Comment:
No because when the objects get recreated we're not creating a player object at the top level and passing it all the way in. We're passing the JSON in from the top and the property say "oh I need to build a player with this data" Personally I don't think a property should have a reference to a player since we talked a bit about it being like a token. Should only be player -> property